ci-operator/jobs/openshift/machine-api-provider-aws/openshift-machine-api-provider-aws-release-4.10-presubmits.yaml

```diff
@@ -75,7 +75,7 @@ presubmits:
     trigger: (?m)^/test( | .* )e2e-aws,?($|\s.*)
   - agent: kubernetes
     always_run: false
-    branches:
+    branches:govet
```

Comment:
why this change? seems like this change is creating CI failure!

Reply:
Sorry, That was not intended, Reverting it back